src/app/app.component.ts

```
@@ -27,60 +29,68 @@
-        'refresh_token': token.refresh_token,
-        'full_name': token.user.full_name,
-        'user_id': token.user.user_id,
-        'org_id': token.user.org_id,
-        'org_name': token.user.org_name
+        'refresh_token': token.refresh_token
       };
-      this.storageService.set('user', user);
-      this.openInAppIntegration(inAppIntegration);
+
+      this.authService.storeTokens(inAppIntegrationKey, tokens);
+      console.log('[x] redirecting from watcher:', inAppIntegrationKey);
```

ESLint

[error] Unexpected console statement.

(no-console)